node/integration_test.go

```diff
@@ -166,7 +167,91 @@ func TestTxGossipingAndAggregation(t *testing.T) {
 	}
 }
 
-func createNodes(aggCtx, ctx context.Context, num int, wg *sync.WaitGroup, t *testing.T) ([]*Node, []*mocks.Application) {
+func TestFraudProofTrigger(t *testing.T) {
+	assert := assert.New(t)
+	require := require.New(t)
+
+	var wg sync.WaitGroup
+	clientNodes := 4
+	aggCtx, aggCancel := context.WithCancel(context.Background())
+	ctx, cancel := context.WithCancel(context.Background())
+	nodes, apps := createNodes(aggCtx, ctx, clientNodes+1, true, &wg, t)
+
+	wg.Add((clientNodes + 1) * clientNodes)
+	for _, n := range nodes {
+		require.NoError(n.Start())
+	}
+
+	time.Sleep(1 * time.Second)
```

Comment:
Document what this sleep is for (and ideally why one second).

Reply:
It's to give nodes some time to start and connect to each other. Value is big to ensure it's not flaky on GitHub CI.